isso/ext/notifications.py

```diff
@@ -139,17 +139,43 @@ def format(self, thread, comment, parent_comment, recipient=None, admin=False):
 
         else:
             uri = self.public_endpoint + "/id/%i" % parent_comment["id"]
-            key = self.isso.sign(('unsubscribe', recipient))
+            key = self.isso.sign(('unsubscribe', recipient["email"]))
```

Comment:
Why is this now looking at ["email"] ? Was the code broken previously?

Reply:
Because I pass the whole comment record of the recipient instead of the email address to it, as you can see below.

Reply:
So the recipient argument is now misnamed?

Reply:
So maybe change it to `recipient_comment`?